docs/syntax-mode.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Syntax Mode Configuration
+---
+
+Flowershow lets you choose how your content is parsed: Markdown or MDX. This is particularly useful when publishing content from Obsidian or other Markdown editors.
+
+## Configuration
+
+### Global Setting
+
+Set the default rendering mode in `config.json`:
+
+```json
+{
+  "syntaxMode": "md"
```

Comment:
BTW does capitalization matter here? I hope not e.g. `syntaxMode` and `syntaxmode` are treated the same ...